app/_kong_plugins/acl/examples/allow-consumers.yaml

```
@@ -0,0 +1,26 @@
+description: 'Allows the `dev` and `admin` Consumers on the Service or Route that the ACL plugin is enabled on.'
+
+title: 'Allow Consumers without apphending the `X-Consumer-Groups` header'
```

```suggestion
title: 'Allow Consumers without appending the X-Consumer-Groups header'
```
The code formatting doesn't work in side nav entries.